packages/create-mechanic/function-templates/svgjs-image/index.js

```diff
@@ -0,0 +1,74 @@
+import { SVG, extend as SVGextend, Element as SVGElement } from '@svgdotjs/svg.js'
```

Comment:
Is this the same dependency listed in the dependencies JSON? It looks like a different name. I tried running it and it can't find that module.